components/org.wso2.carbon.identity.data.publisher.authentication.audit/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/audit/AuthenticationAuditLoggingHandler.java

```diff
@@ -141,7 +144,7 @@ protected void doPublishAuthenticationStepFailure(AuthenticationAuditData authen
 
         AUDIT_LOG.info(String.format(
                 FrameworkConstants.AUDIT_MESSAGE,
-                authenticationData.getAuthenticatedUser(),
+                getUsernameForAuditLog(authenticationData.getAuthenticatedUser()),
```

Comment:
is the user id not available here?

Reply:
Yes, userId is not available as the user is not authenticated yet.